msibi/state.py

```diff
@@ -2,21 +2,39 @@
 
 import mdtraj as md
 
+# **** 2017_05_31 Notes ******
+############### update section start
 
-HOOMD_HEADER = """
+HOOMD1_HEADER = """
 from hoomd_script import *
 
-system = init.read_xml(filename="{0}", wrap_coordinates=True)
```

Comment:
You sure this needs to be deleted? This was working correctly in HOOMD 1.x, right?